Makefile

```diff
@@ -28,5 +28,7 @@ devel:
 clean:
 	rm -rf *.egg-info dist build .pytest_cache */__pycache__ pyarn/parsetab.py pyarn/parser.out
 
-build:
+# Let's run check here and make sure PLY's parsetab is created
+build: clean
```

Comment:
Did you mean to say "check" or "clean"?

Reply:
clean: the idea is to have the script to make sure there the old table (and any other leftovers) were indeed removed to make sure we are starting going for a fresh build

Reply:
IC, it's just that the comment says "check" which is also a make target